Story/Episode 3/np_010851.csv

```diff
@@ -0,0 +1,53 @@
+text02008#0,"""That sounds confusing huh.. To put it simply, Oracle is under my control."""
+text02009#0,"""Though it might get lonely without Xion... I have to push on."""
+11_voice_NPC_5085_100#1,"""For the future of Oracle. For ARKS. For you and me."""
+text03001#0,"""About Darkers... They spread throughout the Universe. An enemy of the Universe, an enemy of every living being."""
+text03002#0,"""Darkers consume all in their path. Every civilization, every living thing."""
+text03003#0,"""Only ARKS can permanently defeat Darkers with the use of Photons."""
+text03004#0,"""Even if someone is able to defeat the Darkers without the use of Photons, the residual fragments will eventually corrupt him. """
+11_voice_NPC_5085_105#0,"""Under the Darkers corruption, they lose any consciousness of themselves, mutating into a ferocious enemy."""
+text03006#0,"""This is not only limited to organic beings. Mechanical beings and Artificial Intelligence are equally susceptible. """
+text03007#0,"""There are variances in mutation, however they end up corrupted all the same. Therefore, only ARKS is able to defeat Darkers."""
+11_voice_NPC_5085_104#1,"""Other means of defeating them may exist, if you consider the endless possibilities of the Universe."""
+text03009#0,"""For now, only ARKS is able to exterminate the Darkers. Which is why we are travelling across the Universe."""
+11_voice_NPC_5085_104#2,"""Dark Falz... To be honest, I do not have much information regarding them... So this will be short."""
+text04002#0,"""Currently, there is 5 confirmed existence of Dark Falz. [Elder], [Apprentice], [Loser], [Double] and lastly, [Persona]"""
+text04003#0,"""I can't say much about their characteristics... Well, you can simply think of them as the Commanders of Darkers."""
+text04004#0,"""[Elder] and [Apprentice] is more active than others in attacking ARKS."""
+text04005#0,"""[Loser] was defeated by Regius.. Due to [Double]'s power however, clones of [Loser] appear frequently."""
+text04006#0,"""As for [Persona]... Honestly, I dont know much."""
+text04007#0,"""Unless we are able to gather more information, there's nothing I can share. Sorry."""
+text04008#0,"""One thing is for sure, we will not have peace until Dark Falz is defeated."""
+11_voice_NPC_5085_100#2,"""It's tough. That's why ARKS has to do it, only ARKS can do it."""
+11_voice_NPC_5085_100#3,"""Matter Board... I can tell you what I understand about it."""
+text05002#0,"""As Xion was capable of calculating every possiblity in this Universe. She was able to calculate her desired outcome."""
+text05003#0,"""<%me>.<br>Matter Board was created for you to change the future."""
+text05004#0,"""Besides making it possible to change the future, you could travel back in time to correct the past."""
+text05005#0,"""Matter Board allowed you to save people that you could not save before, or chancing upon encounters that were not supposed to happen"""
```

Comment:
"The Matter Board"
"or chance upon encounters that were not supposed to happen."